modules/ROOT/pages/index.adoc

```diff
@@ -76,42 +76,42 @@ video::REVkXVxvMQE[youtube]
 
 // link:#[Create a Free Social Sandbox, role=more information]
 
-=== Real-time Recommendations
+=== Movie Recommendations
 
-image:https://neo4jsandbox.com/img/recommendations-icon-pink.png[Create a free Neo4j Movies and Recommendations example datasets]
+image:https://neo4jsandbox.com/img/recommendations-icon-pink.png[Create a Movie Recommendations Sandbox,link="https://sandbox.neo4j.com/?usecase=recommendations&ref=developer-lp"]
```

Comment:
Happy with this line but I wanted to emphasise that sandbox was free.  Maybe there's a better way to communicate that in the copy?